l5kit/l5kit/data/zarr_utils.py

```diff
@@ -222,3 +223,61 @@ def zarr_split(input_zarr: str, output_path: str, split_infos: List[dict]) -> Li
         cur_scene = end_cut
 
     return cuts_track
+
+
+def zarr_scenes_chunk(input_zarr: str, output_zarr: str, num_frames_to_copy: int) -> None:
```

Comment:
What is this function actually for - how about an optional parameter specifying the start scene / frame?

Reply:
desirable, but very risky to implement right now. definitively a TODO though